.gitignore

```diff
@@ -45,7 +45,7 @@ Makefile.in
 
 # build
 .build*
-build
+/build
```

Comment:
What's the difference here?

Reply:
`build` ignores all files named `build` under sub directories.

Reply:
Isn't it what we want?

Reply:
Probably you want it be `build/` to ignore only directories but not files?

Reply:
Actually I see many projects(e.g. CHIP) have a build directory tracked. I think we can only make such assumption for the current directory.